### PR TITLE
Open the browser when razzle is first started

### PR DESCRIPTION
**Pull request by EmilTholin**
It might be a nice addition to open the browser automatically when razzle is first started, like Create React App does.

### CODE REVIEW

File: packages/razzle/scripts/start.js [hunk 1]
```diff
@@ -59,7 +61,13 @@ function main() {
         stats: 'none',
       },
       /* eslint-disable no-unused-vars */
-      stats => {}
+      stats => {
+        // Just open the browser when the watch mode is first activated
+        if (!isInWatchMode) {
+          openBrowser(`http://localhost:${process.env.PORT}`);
```
Comment (olehreznichenko) [inline packages/razzle/scripts/start.js:67]: For example, my razzle project is on `https`. How to handle this case?

Comment (EmilTholin) [reply packages/razzle/scripts/start.js:67]: `localhost` should probably be something like `process.env.HOST || options.host || 'localhost'` instead, but how are you using `https` in development? I didn't know razzle could do that.

Comment (olehreznichenko) [reply packages/razzle/scripts/start.js:67]: https setup was easy.

1. OverridepublicPath for client and server
config.output.publicPath = 'https://localhost:3001/'
2. And for devServer add certificates
```
Object.assign(config.devServer, {
			https: {
				key: readFileSync('./path/localhost.key'),
				cert: readFileSync('./path/localhost.crt')
			}
		})
```
3. Also connect certificates to express or any other server.
Most problem generated for localhost this certificates. and make then trusted for a machine.

Comment (EmilTholin) [reply packages/razzle/scripts/start.js:67]: Alright, that's neat. I was under the impression that setting `process.env.PUBLIC_PATH` or changing `webpack.config.output.publicPath` was only supported in the build script.

Will have to think about the protocol issue some more.

Comment (olehreznichenko) [reply packages/razzle/scripts/start.js:67]: Maybe add new env PROTOCOL?

**Comment by MioQuispe** [comment]
Could this be done as a plugin maybe?

**Comment by EmilTholin** [comment]
Yeah, that's a good idea. Not optimal to force the browser opening on users.

**Comment by joeferraro** [comment]
Was this ever implemented as a plugin?

**Pull request closed without merging**